src/libwhb/src/log.c

```diff
@@ -58,57 +58,78 @@ BOOL
 WHBLogPrint(const char *str)
 {
    char *buf = MEMAllocFromDefaultHeapEx(PRINTF_BUFFER_LENGTH, 4);
-   BOOL result;
-
    if(!buf) {
       return FALSE;
    }
 
    snprintf(buf, PRINTF_BUFFER_LENGTH, "%s\n", str);
-   result = WHBLogWrite(buf);
+
+   int i;
```

Comment:
Instead of repeating this in all the log functions maybe split it out to a static function?

Like
```
static void
dispatchLogMessage(const char *msg)
{
   int i;
   for (i = 0; i < MAX_HANDLERS; ++i) {
      if (sHandlers[i]) {
         sHandlers[i](msg);
      }
   }
}
```

```
WHBLogPrint()
{
...
dispatchLogMessage(buf);
...
}
```